envs/monkey_zoo/blackbox/island_client/agent_requests.py

```diff
@@ -0,0 +1,83 @@
+from http import HTTPStatus
+from typing import Dict
+
+import requests
+
+from common.types import OTP, AgentID
+
+from .i_monkey_island_requests import IMonkeyIslandRequests
+
+
+class InvalidRequestError(Exception):
+    pass
+
+
+class AgentRequests(IMonkeyIslandRequests):
```

Comment:
Should we just use `infection_monkey.island_api_client.IIslandAPIClient`?

Pros:
- Less code in BB tests
- Adds a quick, ETE test for HTTPIslandAPIClient

Cons: 
- Couples the tests to that component

Reply:
I don't like it. These tests shouldn't know anything about the internals. We don't do anything like this anywhere else in the BB tests either.